test/factories/projects.rb

```diff
@@ -30,9 +30,14 @@
   f.documents {[Factory(:document, policy: Factory(:public_policy))]}
   f.workflows {[Factory(:workflow, policy: Factory(:public_policy))]}
   f.after_create do |p|
-    Factory(:person, project: p)
+    member = Factory(:person, project: p)
     p.reload
     p.default_policy = Factory(:private_policy)
+
+    User.with_current_user member.user do
+      p.edam_topics = ['Biomedical science','Chemistry']
+    end if SampleControlledVocab::SystemVocabs.edam_topics_controlled_vocab
+
```

Comment:
yes they do, I decided from a previous pull request to save generating those to particular milestones, and generate outside of the PR - otherwise you end up with over about 100 changed files making it harder to review

Reply:
e.g. #1028